src/node/internal/streams_util.js

```diff
@@ -459,8 +459,9 @@ export function highWaterMarkFrom(options, isDuplex, duplexKey) {
       : null;
 }
 
+// Node.js uses 16 * 1024 on Windows, and 64 * 1024 on rest of the platforms.
```

Comment:
nit: this may look like a bug and you wanted to write `(objectMode ? 16 : 64) * 1024`

Maybe clarify

Reply:
Yeah, the comment is wrong, I think. Also need to make sure the actual value is correct. The default highWaterMark in `objectMode` is 16. The default highWaterMark when `!objectMode` is 64 * 1024.